test/attributes.test.ts

```diff
@@ -431,36 +380,28 @@ describe("Read Attribute and versions", () => {
             }
         });
 
-        await client1.attributes.notifyPeerAboutRepositoryAttributeSuccession(succeededAttributeResponse.result.predecessor.id, {
-            peer: client2Address
-        });
-
-        const ownAttributesResponse = await client1.attributes.getOwnSharedIdentityAttributes({
-            peer: client2Address,
-            onlyLatestVersions: true
-        });
-
-        const peerAttributesResponse = await client2.attributes.getPeerSharedIdentityAttributes({
-            peer: client1Address,
-            onlyLatestVersions: true
-        });
+        const result = await client1.attributes.notifyPeerAboutOwnIdentityAttributeSuccession(succeededAttributeResponse.result.successor.id, { peer: client2Address });
+        expect(result).toBeSuccessful();
 
+        const ownAttributesResponse = await client1.attributes.getOwnAttributesSharedWithPeer({ peer: client2Address, onlyLatestVersions: true });
```

Comment:
Can be omitted as you do below.

```suggestion
        const ownAttributesResponse = await client1.attributes.getOwnAttributesSharedWithPeer({ peer: client2Address });
```